server/views/pages/explore.njk

```diff
@@ -28,6 +28,22 @@
     </div>
   </div>
 
+  <div class="row {{ {s:2, m:2, l:2} | spacingClasses({padding:['top']}) }}">
+    <div class="container container--scroll-xl">
+      <div class="grid grid--no-gutters">
+        <div class="{{ {s: 4, m: 8, l: 12, xl: 12} | gridClasses }}">
+          {% set asyncContentEndpoint = '/series-transporter/' + latestDigitalStory %}
+          {% component 'async', { endpoint: asyncContentEndpoint, component: 'series-transporter' } %}
```

Comment:
That `slider-transporter-child` is a modifier for the promo which overrides the [promo's container-query behaviours](https://github.com/wellcometrust/wellcomecollection.org/pull/911/files/73ae975ea95bf4039084cefe8df19a0a7da84ffc#diff-106f0f02dce48cb540307355e721b7e0R386)